src/dguweb/web/router.ex

```diff
@@ -45,7 +46,10 @@ defmodule DGUWeb.Router do
   end
 
   # Other scopes may use custom stacks.
-  # scope "/api", DGUWeb do
-  #   pipe_through :api
-  # end
+  scope "/api", DGUWeb do
+     pipe_through :api
+
+     get "/autocomplete", AutocompleteController, :query
+
```

Comment:
Because you can't test by pasting it in a browser. And it screws up varnish caching. And whenever I've seen Accept headers tried for linked data things, I've seen users scratching their heads (even more than usual for linked data). I realize they are the HTTP way, but I'd urge us to avoid them, without compelling reasons.